modules/gdscript/language_server/gdscript_text_document.cpp

```diff
@@ -346,6 +346,15 @@ Dictionary GDScriptTextDocument::resolve(const Dictionary &p_params) {
 		}
 	}
 
+	if (item.kind == lsp::CompletionItemKind::Method) {
+		bool isTriggerCharacter = params.context.triggerKind == lsp::CompletionTriggerKind::TriggerCharacter;
+		bool isQuoteCharacter = params.context.triggerCharacter == "\"" || params.context.triggerCharacter == "'";
+
+		if (isTriggerCharacter && isQuoteCharacter && item.insertText.is_quoted()) {
+			item.insertText = item.insertText.unquote();
+		}
```

Comment:
```suggestion
		bool is_trigger_character = params.context.triggerKind == lsp::CompletionTriggerKind::TriggerCharacter;
		bool is_quote_character = params.context.triggerCharacter == "\"" || params.context.triggerCharacter == "'";

		if (is_trigger_character && is_quote_character && item.insertText.is_quoted()) {
			item.insertText = item.insertText.unquote();
		}
```
Use `snake_case` in c++